compiler/src/dotty/tools/dotc/transform/localopt/StringContextChecker.scala

```diff
@@ -49,7 +49,7 @@ object StringContextChecker {
       *
       * @return true if an error/warning has been reported, false
       */
-    def hasReported() : Boolean
+    def hasReported : Boolean
```

Comment:
This one might be reverted to minimize diff

Reply:
That was my favorite itch to scratch.